.github/workflows/hil_tests.yml

```
@@ -247,6 +247,21 @@ jobs:
             twister-out/*.xml
             twister-out/*.json
 
+      - name: Prepary report summary
+        if: always() && steps.mask-logs.outcome == 'success'
+        run: |
+          mkdir summary
+          cp twister-out/twister_suite_report.xml summary/twister-${{ matrix.artifact_suffix }}.xml
+
+      - name: Upload report summary
+        uses: actions/upload-artifact@v4
+        if: always() && steps.mask-logs.outcome == 'success'
+        with:
+          name: ci-summary-twister-${{ matrix.artifact_suffix }}
```

```suggestion
          name: ci-summary-samples-zephyr-${{ matrix.artifact_suffix }}
```
Use an artifact title that more closely follows the workflow names